2-mandatory/3-financial-times.js

```diff
@@ -5,7 +5,14 @@
     Implement the function below, which will return a new array containing only article titles which will fit.
 */
 function potentialHeadlines(allArticleTitles) {
-    // TODO
+  const rightTitles = [];
+
+  for (titles of allArticleTitles) {
```

Comment:
Good implementation :) One minor point is the naming of the `titles` variable. Each time through the loop this will be assigned to one title - so a better name for this would be `title`.